Classes/common/HTTP/CDTURLSessionInterceptorContext.m

```diff
@@ -0,0 +1,29 @@
+//
+//  CDTURLSEssionFilterContext.m
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import "CDTURLSEssionInterceptorContext.h"
+
+@implementation CDTURLSessionInterceptorContext
+
+-(instancetype) init {
```

Comment:
My understanding is it should call into `initWithRequest:nil` which may choose to return `nil`.